### ISSUE TITLE
Related Links insert invalid data

### ISSUE THREAD

**Issue by nul800sebastiaan** [KevinJump/LocalGovStarterKit#3]
When installing the starterkit on a clean Umbraco 7.3.4, the sample content adds a few properties of type `Umbraco.RelatedLink`. The data in `cmsPropertyData` looks like this:

```
[
  {
    "caption": "Starter Kit Help",
    "link": 8B40D7C0-E98E-49C7-BED3-B1DAFD92FAF7,
    "newWindow": false,
    "internal": 8B40D7C0-E98E-49C7-BED3-B1DAFD92FAF7,
    "edit": false,
    "isInternal": true,
    "internalName": "Help",
    "type": "internal",
    "title": "Starter Kit Help"
  }
]
```

This is not only invalid JSON, as the quotes around the GUIDs are missing but most of all this should've been integer Ids. 
It could well be that we broke something in the core that uSync relies upon. If this is the case then I'd love to know so we can look at the core to see if we can fix something there.

**Comment by KevinJump** [comment]
yeah uSync should turn the guids into IDs at some point :( 

i think it's much more likely i've broken the way the starter kit uses the usync core to load it's example content.

**Comment by KevinJump** [comment]
I think it's a missing config file - in the package. 

the package contains the wrong version of uSyncCore.config, so doesn't know that it needs to map RelatedLink Ids when importing. putting the right config in the package will hopefully fix that - 

it's not umbraco :smile:

**Comment by nul800sebastiaan** [comment]
:+1: good to know! :)

**Comment by KevinJump** [comment]
fixed (and released) in 5.1.3
